core/services/blockhashstore/coordinators.go

```diff
@@ -72,7 +71,7 @@ type V1Coordinator struct {
 
 // NewV1Coordinator creates a new V1Coordinator from the given contract.
 func NewV1Coordinator(c v1.VRFCoordinatorInterface, lp logpoller.LogPoller) (*V1Coordinator, error) {
-	err := lp.RegisterFilter(logpoller.Filter{
+	err := lp.RegisterFilter(context.Background(), logpoller.Filter{
```

Comment:
Is there a ctx that can be passed by `NewV1Coordinator`'s caller instead?

Reply:
Yeah for sure